catalogs/catalogio.py

```diff
@@ -126,11 +126,13 @@ def get_next_control_by_id(self, control_id: str):
         try:
             next_idx = search_collection.index(control_id) + 1
             return search_collection[next_idx]
-        except ValueError as exc:
-            raise MissingControlError(
-                "Cannot determine next control. Provided id does not exist"
-            ) from exc
+        except ValueError:
+            logger.warning(
+                "Cannot determine next control. Provided control does not exist in catalog(s): %s", control_id
+            )
+            return ""
         except IndexError:  # control_id was at the end of the list
+            logger.info("No new controls.")
```

Comment:
Apparently it's allowed for components to be defined with controls that might not be in the catalogs..